CHANGELOG.md

```diff
@@ -7,6 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.79.0] - 2023-05-25
+
+### Added
+
+1. Feat add status bar in ionic page between image and caption (#2810)
+
+### Fixed
+
+1. Refactor pre publish component (#2809)
+1. Fix issue flip camera for oppo and other devices if any (#2811)
+1. Fix issue error message when without device permission (#2812)
+
+## [0.78.0] - 2023-04-24
```

Comment:
I forgot to add 0.78.0 changelog previously so I added in this commit with 0.79.0 changelog update.

P.S: 0.78.0 is milestone-v230321